iothub/service/src/Configurations/ConfigurationImportMode.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Identifies the behavior when merging a configuration to the registry during import actions.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ConfigurationImportMode
+    {
+        /// <summary>
+        /// If a configuration does not exist with the specified Id, it is newly registered.
+        /// If the configuration already exists, existing information is overwritten with the provided input data only if there is an ETag match.
+        /// If there is an ETag mismatch, an error is written to the log file.
+        /// </summary>
+        [EnumMember(Value = "createOrUpdateIfMatchETag")]
+        CreateOrUpdateIfMatchETag = 4,
```

Comment:
Isao says they only support this mode for now, so we're not using the same `ImportMode` class as is used for ExportImportDevice, but I matched the same enum value as the other enum to allow room for the other items to be inserted above (numerically).